app/src/main/java/edu/uc/zhukv/droneradarmap/MapsActivity.kt

```diff
@@ -58,7 +63,7 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
             val location = mFusedLocationProviderClient.lastLocation
             location.addOnCompleteListener { task ->
                 if (task.isSuccessful) {
-                    var currentLocation:Location? = task.result
+                    val currentLocation:Location? = task.result
```

Comment:
naming conventions of the the varaible - https://kotlinlang.org/docs/coding-conventions.html#property-names